rpcserver.go

```diff
@@ -6601,11 +6599,12 @@ func marshalPeerAcceptedSellQuotes(quotes map[rfq.SerialisedScid]rfqmsg.SellAcce
 			Scale:       uint32(quote.AssetRate.Scale),
 		}
 
+		// TODO(ffranr): Add SellRequest payment max amount to
+		//  PeerAcceptedSellQuote.
```

Comment:
shouldn't this TODO be addressed before merging?

Reply:
I'll put up another PR to address this. This PR is getting a bit long.